packages/shorebird_cli/test/src/extensions/string_test.dart

```diff
@@ -7,4 +7,11 @@ void main() {
     expect(''.isNullOrEmpty, true);
     expect('test'.isNullOrEmpty, false);
   });
+
+   test('ToKebabCase', () async {
```

Comment:
What does this do for text containing spaces?

Reply:
it wont make any change as of now, will make changes to convert space to hypen (-).